LICENSE

```diff
@@ -1,3 +1,8 @@
+==============================================================================
```

Comment:
Does this addition break github's license detection algorithm?  I believe it checks whether the file is identical (or almost identical) to a list of known licenses: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#detecting-a-license

Reply:
Good question, it doesn't look like it: https://github.com/Kha/lean4
So no header for LICENSE I suppose